presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -37,6 +37,15 @@ std::string bool2String(bool value) {
   return value ? "true" : "false";
 }
 
+int getThreadCount() {
+  auto numThreads = std::thread::hardware_concurrency();
+  // The spec says std::thread::hardware_concurrency() might return 0.
+  if (numThreads <= 0) {
+    numThreads = 16;
```

Comment:
Nit : Add static constant for this number.

Reply:
This function is in an anonymous namespace. I don't think there is any advantage of using a constant here. All the default config values use literals for example.

Reply:
@majetideepak : All of the SystemConfig property defaults are either in the registeredProps_ map (where static constant isn't needed) or they have individual SystemConfig functions to retrieve the properties if there is some logic like this which could use a default value. They aren't in anonymous namespace functions.

We should be consistent with that in general.

Reply:
Using an anonymous namespace is equivalent to using static.  https://stackoverflow.com/questions/4422507/how-are-unnamed-namespaces-superior-to-the-static-keyword
I can add a constant.

Reply:
@majetideepak : Thanks for that article. Agree that we should reduce our use of static constants and functions as much as possible for external linkage.

Made the review comment largely because of consistency with the rest of this class.

Reply:
@aditi-pandit I ended up removing the constant. I added a check instead.